### ISSUE TITLE
failed to compile on msys2

### ISSUE THREAD

**Issue by iahung** [dictu-lang/Dictu#568]
by msys2 I mean the mingw64 shell. As I recall there used to be an issue for this, but now it's disappeared. anyway, the problem is still there.

**Comment by Jason2605** [comment]
Yeah I’m not entirely sure what happened to that issue now you’ve mentioned it

**Comment by iahung** [comment]
> Yeah I’m not entirely sure what happened to that issue now you’ve mentioned it

but the problem is still there, I just tried to compile and failed.

**Comment by Jason2605** [comment]
Will need to investigate this. Will need to check what my windows machine I've been testing with compiles with
